refactor: make dropdowns uniform

jniles opened the pull request:
Goes through all dropdowns in the application and makes them uniform by aligning them to the right (`.text-right`) and also making sure they don't overflow but instead use `text-ellipsis` to overflow.  I've also put headers on the dropdown menus that do not have headers and expanded the text of those where it would be more useful.

Closes #4586.
Closes https://github.com/IMA-WorldHealth/bhima/issues/4587

jniles committed:
refactor: make dropdowns uniform

Goes through all dropdowns in the application and makes them uniform by
aligning them to the right (`.text-right`) and also making sure they
don't overflow but instead use `text-ellipsis` to overflow.  I've also
put headers on the dropdown menus that do not have headers and expanded
the text of those where it would be more useful.

Closes #4586.

mbayopanda commented on client/src/modules/account_reference/templates/action.cell.html:
```diff
@@ -6,7 +6,7 @@
 
   <ul data-row-menu="{{row.entity.abbr}}" class="dropdown-menu-right" bh-dropdown-menu-auto-dropup uib-dropdown-menu>
-    <li class="dropdown-header text-ellipsis" style="max-width:250px;">{{row.entity.abbr}}</li>
+    <li class="bh-dropdown-header">{{row.entity.abbr}}</li>
```
👍

mbayopanda commented on client/src/modules/dashboards/indicators_files_registry/indicators_files_registry.js:
```diff
@@ -13,7 +13,7 @@ function IndicatorsFilesRegistryController(
   const vm = this;
 
-  const cacheKey = 'AdmissionRegistry';
+  const cacheKey = 'IndicatorFilesRegistry';
```
👍

mbayopanda commented on client/src/modules/employees/templates/action.cell.html:
```diff
@@ -5,7 +5,7 @@
 
   <ul data-row-menu="{{ row.entity.reference }}" class="dropdown-menu-right" bh-dropdown-menu-auto-dropup uib-dropdown-menu>
-    <li class="dropdown-header">{{row.entity.reference}}</li>
+    <li class="bh-0dropdown-header">{{row.entity.reference}}</li>
```
`bh-dropdown-header` i think

mbayopanda reviewed (commented):
There are a lot of file changed 😄 

The use of ellipsis is very nice, i just discover for some registries there also divider under the `bh-dropdown-header` for example : 
- Administration -> Configuration des outils d'analyse
- Administration -> Reference de compte
- Administration -> Reference Seuil de rentabilite
- Collecte de données -> Kit de collecte de données ->  Configuration des formulaires
- Collecte de données -> Kit de collecte de données -> Gestion des formulaires
- Collecte de données -> Kit de collecte de données -> Gestion des listes de choix

For the account module, the ellipsis doesn't work we have something like : 
![image](https://user-images.githubusercontent.com/5445251/84300278-fd5e4f00-ab49-11ea-92e9-f86117a3fb84.png)

Instead of having something like : 
![image](https://user-images.githubusercontent.com/5445251/84300384-1f57d180-ab4a-11ea-9bdf-81dd16a40d66.png)

- Registre des employes there is a `bh-0dropdown-header` instead of `bh-dropdown-header`

jniles committed:
chore(dropdown): remove dividers under headers

Removes the `<li class="divider">` appearing directly under the header
of the dropdown for better uniformity.

mbayopanda commented on client/src/less/bhima-bootstrap.less:
```diff
@@ -539,3 +539,16 @@ div.ui-grid-cell .form-group.has-error input.ng-invalid {
+	line-height: 1.42857143;
+  max-width:250px;
+	overflow: hidden;
+	padding: 3px 20px;
```
👍

mbayopanda reviewed (approved):
bors r+
